tempel-collections.el

```diff
@@ -26,6 +26,7 @@
 ;;; Commentary:
 ;; This package adds support for templates using tempel.  It will add templates from
 ;; template directory to tempel-path.
+;; Based on mpenet/tempel-closure project autored by Max Penet <mpenetr@s-exp.com>
```

Comment:
Thanks, 


There are a few typos but otherwise that's fine. Here is an amended version that fixes these

> ;; Based on https://github.com/mpenet/tempel-clojure authored by Max Penet <mpenet@s-exp.com>

Reply:
Done

Reply:
perfect. thanks!